src/main/scala/testing/InstTests.scala

```diff
@@ -30,71 +32,85 @@ object InstTests {
 	val rtype = List[CPUTestCase](
 		CPUTestCase("add1",
                 Map("single-cycle" -> 1, "pipelined" -> 5),
+								Map(), Map(),
```

Comment:
Did you mean to remove this?

Reply:
yup. fixed. remove csr parameters from testing